recipe/meta.yaml

```diff
@@ -9,14 +9,14 @@ source:
   sha256: 3ba0b78b93cb69d8c865b9cdcde57be2da9b174096a1c075e6fc9188c8f8afd1
 
 build:
-  number: 1002
+  number: 1003
```

Comment:
@ocefpaf What's the reason behind the `1000` offset for the build number? Just curious...

Reply:
Everything that was migrated to the new compilers is +1000 and is published on the `gcc7` channel, once we "flip" the switch they will be moved to the `main` channel and the build number will be current `-1000`.

Reply:
So if the version number is bumped up, the build number is reset back to `1000` instead of `0`, right?

Reply:
Yep. (Only during this transition, once we flip the switch on the migration this will be back to normal.)

Reply:
Sweet, thanks as always @ocefpaf

Reply:
... (sorry) how long's the transition period? Is this documented or easily discoverable?

I'm really expecting you to say no here :wink:

Reply:
> I'm really expecting you to say no here

Thanks b/c that is the answer. Dunno and no :smile: 

PS: we do have a status page for the migration though.

https://conda-forge.org/status/